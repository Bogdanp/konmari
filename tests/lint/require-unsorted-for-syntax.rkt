#lang racket/base

(require racket/string
         (for-syntax racket/base)
         racket/format
         racket/file
         "b.rkt"
         "a.rkt")
