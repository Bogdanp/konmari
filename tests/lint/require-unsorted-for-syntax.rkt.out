/require-unsorted-for-syntax.rkt:4:10:warning:require (for-syntax ...) should come before all others
/require-unsorted-for-syntax.rkt:6:10:warning:require "racket/file" should come before "racket/format"
/require-unsorted-for-syntax.rkt:8:10:warning:require "a.rkt" should come before "b.rkt"
